### PR TITLE
Feature/run tests

### PR DESCRIPTION
**Pull request by stanleyzhengnyc**

### CODE REVIEW

File: .github/.workflows/verify.yaml [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+      - name: Install packages
+        run: |
+          yarn install --pure-lockfile
+
+      - name: Run linters
+        run: |
+          bin/rubocop --parallel
```
Comment (jtmckibb) [inline .github/.workflows/verify.yaml:48]: nice

File: .github/.workflows/verify.yaml [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+          bin/stylelint
+          bin/prettier
```
Comment (jtmckibb) [inline .github/.workflows/verify.yaml:50]: didn't know prettier/eslint/stylelint existed for ruby

Comment (stanleyzhengnyc) [reply .github/.workflows/verify.yaml:50]: i dont either i just hope it works

Comment (Yong-L) [reply .github/.workflows/verify.yaml:50]: LOL

**Pull request merged**